### ISSUE TITLE
Virtual Bags

### ISSUE THREAD

**Issue by mkaito** [ForgeUI/ForgeUI#187]
Yes, I know Kurona does this, but I find the candy look of Kurona Bags quite jarring. Any chance of implementing some of that functionality in ForgeUI?

**Comment by adamjedlicka** [comment]
Yeah, I want to do this, but I don't have any time at all lately, so doubt I will be able to do this any time soon.

**Comment by mkaito** [comment]
It's probably not trivial to implement either, but it'd be pretty awesome. Just leave the issue open until you find the time and motivation, or perhaps someone else can try to implement it.
